portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,37 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Kevin's Portfolio</h1>
+      <img id="portrait" src="images/portrait.jpg">
+      
+      <h3>About Me</h3>
+      <p> Click here to get a random fact about me</p>
+      <button onclick="addRandomFact()">Random Facts About Me</button>
+      <div id="fact-container"></div>
+      <p>I grew up in Miami, Florida surrounded by a Latinx community and playing video games such as Starcraft 2 and Call of Duty. 
+          I am currently a rising Sophomore at Northeastern University studying computer science. 
+          During my time at Northeastern I joined a salsa/bachata dance club called Kaliente in order to maintain some resemblace of home while I was away studying in Boston. 
+          Growing up in Miami, a city centered around cars, I am fascinated with the urban-planning model called superblocks that are being implemented in Spain! 
+          I hope to one day travel to Spain and experience their culture rich cities.
```

Comment:
Try to wrap all lines to either 80 or 100 characters so they can fit within a standard editor window